src/components/Balance.module.css

```diff
@@ -1,9 +1,14 @@
 .balance-wrapper {
   display: inline-flex;
-  align-items: baseline;
+  align-items: center;
+
+  min-height: 23px;
```

Comment:
I really do not know if it is this value that causes the misbehaviour: In Chromium (v97) and in Brave (v1.33, based on Chromium v96) the wrapper div changes it size when toggling `showBalances` on the settings screen. Does not happen in Firefox (v95). Can you reproduce this?

Reply:
Good find! The sat symbol and the "balance hidden" symbol had different heights. Fixed! ✅